cpp/src/parquet/bloom_filter_writer.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "bloom_filter.h"
+
+#include <map>
+#include <vector>
+
+namespace parquet {
+
+class PARQUET_EXPORT RowGroupBloomFilterReference {
+ public:
+  struct Reference {
```

Comment:
I think there is already a structure we are using for indexing and in other places of offset and length, lets please use that?

Reply:
Sure, we can reuse same structure